README.md

```diff
@@ -73,7 +73,7 @@ limitations under the License.
 [travis]: https://travis-ci.org/snowplow/snowplow-mini
 [travis-image]: https://travis-ci.org/snowplow/snowplow-mini.svg?branch=master
 
-[release-image]: http://img.shields.io/badge/release-0.7.0-blue.svg?style=flat
+[release-image]: http://img.shields.io/badge/release-1.0.0-blue.svg?style=flat
```

Comment:
Is it 0.8 or 1.0 ?

Reply:
nice catch, thanks @benjben ! it is 0.8.0, I'll fix it